recipes/lyteidl/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "lyteidl" %}
+{% set version = "0.2.1" %}
+{% set python_max = "3.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/lyteidl-{{ version }}.tar.gz
+  sha256: 9c5ebe96615e4fd03dd393c49f6356285be8e3860c58271ac7c8babc170165a8
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools
+  run:
+    - python >={{ python_min }},<={{ python_max }}
+    - googleapis-common-protos
+    - protoc-gen-swagger
+    - protobuf >=3.5.0,<4.0.0
+    - typing >=3.6  # [py==27]
```

Comment:
Just remove `typing` since it isn't needed for the versions of python that are specified.